UI/AppKit/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+set(CMAKE_OSX_DEPLOYMENT_TARGET 14.0)
```

Comment:
The availability fix looks reasonable, but I'm not sure this is the correct location to set the deployment target.

Should we not set it globally for all of our libraries and executables?

.. Given how our cmake initialization is set up, it should be somewhere in common_options.cmake, before including swift-settings.cmake. swift-settings.cmake checks this value to ensure consistency between compilers.

Reply:
That probably makes more sense, I don't have a good grasp on the CMake setup we're using, so I'm super happy to let you point me in a better direction. 😄